libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -697,11 +697,34 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 			// Callee removes them and pushes return values
 
 			evmasm::AssemblyItem returnLabel = m_context.pushNewTag();
-			for (unsigned i = 0; i < arguments.size(); ++i)
-				acceptAndConvert(*arguments[i], *function.parameterTypes()[i]);
+
+			if (!_functionCall.isSuffixCall() || parameterTypes.size() == 1)
```

Comment:
The negated condition ready much easier and you can just flip the blocks below...

Reply:
Negated.